Blog: Monitoring System Health and Performance at Scale with FlowFuse

sumitshinde-84 opened the pull request:
## Description

<!-- Describe your changes in detail -->

## Related Issue(s)

<!-- What issue does this PR relate to? -->

## Checklist

<!-- https://flowfuse.com/handbook/development/#defining-done -->

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

sumitshinde-84 commented:
Hey @joepavitt, would you be able to review it today?

joepavitt reviewed (changes requested):
It's very word-heavy in the first-half. I've made some aesthetic improvements, but I've also put in placeholder comments where I think screenshots would be good.

sumitshinde-84 commented:
@joepavitt Could we get this merged by today

joepavitt commented:
I added in a bunch of areas requested screenshots, and that hasn't yet been actioned:

<img width="916" alt="Screenshot 2025-02-21 at 14 17 27" src="https://github.com/user-attachments/assets/8c8320a5-e8c5-4492-af44-ca4c70adf4e4" />

sumitshinde-84 commented:
> I added in a bunch of areas requested screenshots, and that hasn't yet been actioned:
> 
> <img alt="Screenshot 2025-02-21 at 14 17 27" width="916" src="https://private-user-images.githubusercontent.com/99246719/415684015-8c8320a5-e8c5-4492-af44-ca4c70adf4e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNDkyMjQsIm5iZiI6MTc0MDE0ODkyNCwicGF0aCI6Ii85OTI0NjcxOS80MTU2ODQwMTUtOGM4MzIwYTUtZThjNS00NDkyLWFmNDQtY2E0YzcwYWRmNGU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE0NDIwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwYWZlNWUxYmVlYTgwYWJlYjBmZTVlMjM4ODA5YTg0NDkwMjA1ODJjNzI2ZDNlNjY1OWNlZTVlNTkwZDI0Y2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.griROaC29S3tPCYa9BTivYQuSTvl_ExSq9HOUlNshzY">

I have already added those screenshots. Are you seeing the latest preview?
